subsys/zbus/Kconfig

```diff
@@ -79,11 +79,14 @@ config ZBUS_ASSERT_MOCK
 
 
 config HEAP_MEM_POOL_ADD_SIZE_ZBUS
-	int
+	int "Zbus message subscriber heap pool size."
 	default 2048 if ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_DYNAMIC && !ZBUS_RUNTIME_OBSERVERS
 	default 1024 if !ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_DYNAMIC && ZBUS_RUNTIME_OBSERVERS
 	default 3072 if ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_DYNAMIC && ZBUS_RUNTIME_OBSERVERS
-
+	default 0
+	help
+	  This configures heap memory pool available for configured net buffers via ZBUS_MSG_SUBSCRIBER_NET_BUF_POOL_SIZE.
+	  Can be set, if more heap is required, or default heap size is too big for the soc.
```

Comment:
Please make that more generic since zbus also uses heap for other means. You can comment on that to state the importance of this configuration.